src/main/java/meteordevelopment/meteorclient/events/entity/DamageEvent.java

```diff
@@ -13,12 +13,12 @@ public class DamageEvent {
 
     public LivingEntity entity;
     public DamageSource source;
-    public float amount;
+    @Deprecated(forRemoval = true)
+    public final float amount = 0;
```

Comment:
just remove this so addons using this variable will crash

Reply:
i was planning on keeping this for a major version change, but 1.20.2 came out so probably